### DISCUSSION TITLE
Unable to save the filter configuration changes

### DISCUSSION THREAD

**Discussion by athilikarthik** [microsoft-search/pnp-modern-search#3303] [Q&A] [answered]
Hello Everyone, 

I have configured the PnP search webparts on the SharePoint page. The filter web part is configured to have 5 refiners. However, while I am trying to change the property mapping of one of the refiners, the changes do not get saved and reflected. The old value continues to persist.

Please let me know your suggestions. I can remove the web part and reconfigure, but I think it is not the solution as it might happen again. 

Thanks
Regards

**Comment by kasperbolarsen** [comment]
Could you check that the new property you try to set actually is listed in the Selected properties in the results web part?

**Reply by athilikarthik** [reply]
Yes, the property I am trying to set is available in the results web part. I have checked all the other properties and seems all of them are available.

**Comment by athilikarthik** [comment, marked as answer]
I have removed the existing filter property and added the new one. It got saved. I am not sure why it was getting save while modifying the existing one.
